README.md

```diff
@@ -49,249 +53,6 @@ print(pub)
 print([citation.bib['title'] for citation in pub.get_citedby()])
 ```
 
-## Methods for `scholar`

```

Comment:
You can generate the documentation by going to the docs folder and do `make html`
you just need to install sphinx

Reply:
Can you add this step also in the build file for Github actions?

https://github.com/scholarly-python-package/scholarly/blob/master/.github/workflows/pythonpackage.yml

Reply:
Do you think its really necessary?
I have already setup a webhook that builds the documentation here: https://scholarly.readthedocs.io/en/latest/

Reply:
If you have a webhook, then I do not think it is strictly necessary, but it would be good to have a quick check that the documentation builds without issues.